packages/paste-website/src/pages/components/tooltip/index.mdx

```diff
@@ -202,9 +200,53 @@ For full details on how to use the state hook, and what props to provide it, fol
   {StateHookTabsExample}
 </LivePreview>
 
-## Anatomy
+## Composition notes
+
+Tooltips should contain only information that is useful additional context or guidance for the user. Furthermore, Tooltips are easily overlooked and disappear when not focused on, so consider whether the information may be better served by another component.
+
+Never put information essential to task completion in a Tooltip. Use [Help Text](/components/help-text), which is persistent and more obvious, for essential information.
+
+Do not use a Tooltip on a disabled [Button](/components/button) to explain why the button is unavailable. The disabled button isn't focusable. 
+
+Similarly, do not including an [Anchor](/components/anchor) in Tooltip content because an Anchor isn't focusable. If you need to include a link, use a [Popover](/components/popover) instead.
+
+Use full sentences and punctuation. Include no more than 3 lines of text. Avoid titles and icons.
 
-### Tooltip
+## Do and don't
+
+<DoDont>
```

Comment:
chore: Include the Do and Don't component imports at the top of the file so that the Dos and Don'ts show up on the page.